components/opponent/commons/opponent_display.lua

```diff
@@ -98,9 +98,10 @@ end
 OpponentDisplay.propTypes.InlineOpponent = {
 	flip = 'boolean?',
 	opponent = MatchGroupUtil.types.GameOpponent,
-	showFlag = 'boolean?',
-	showLink = 'boolean?', -- does not affect opponent.type == 'team'
-	teamStyle = TypeUtil.optional(OpponentDisplay.types.TeamStyle),
+	showFlag = 'boolean?', -- only affects opponent.type == 'solo'
+	showLink = 'boolean?', -- only affects opponent.type == 'solo'
+	dq = 'boolean?', -- only affects opponent.type == 'solo'
```

Comment:
the comments are not entirely correct
for duo, trio and quad opponents it should work too iirc (when they get added via /Custom)

Reply:
The code here specifically check's solo, so I'd say it's OK